Dockerfile

```diff
@@ -9,28 +9,47 @@ WORKDIR /app
 VOLUME /app/outputs
 
 RUN apt-get update && \
-		apt-get install -y --no-install-recommends \
-		build-essential \
-		git && \
-		rm -rf /var/lib/apt/lists/*
+    apt-get install -y --no-install-recommends \
+        cmake \
+        pkg-config \
+        libssl-dev \
+        build-essential \
+        ca-certificates \
+        git && \
+    rm -rf /var/lib/apt/lists/* && \
+    update-ca-certificates
 
 RUN pixi install --frozen -e ${PIXI_ENV}
 
-RUN pixi run -e ${PIXI_ENV} playwright install
+RUN pixi run --frozen -e pbuild build-wheels
+# RUN pixi add --frozen pip
+# RUN pixi run --frozen -e ${PIXI_ENV} \
+#     python3 -m pip install --no-deps --disable-pip-version-check \
+#     dist/nrel_compass-*.whl
+# Do we want to remove pip to reduce production's size?
+# RUN pixi remove --frozen -e ${PIXI_ENV} pip
+
 
 RUN pixi shell-hook -e ${PIXI_ENV} -s bash > /shell-hook
 RUN echo "#!/bin/bash" > /app/entrypoint.sh
 RUN cat /shell-hook >> /app/entrypoint.sh
 RUN echo 'exec "$@"' >> /app/entrypoint.sh
 
+# ==== Production ====
 FROM ubuntu:24.04 AS production
 
 COPY --from=build /app/.pixi/envs/default /app/.pixi/envs/default
+# Installed wheels, thus we don't need to copy the source anymore
```

Comment:
We can remove this comment until we actually use the wheels, to avoid confusion.

Reply:
Sounds good. If you don't mind, I will do that in a separate PR to avoid re-running the tests, which can't use the cache effectively right now